ecs-cli/modules/cli/compose/containerproject/ecscliservconfig/ecscliservconfig.go

```diff
@@ -0,0 +1,18 @@
+package ecscliservconfig
```

Comment:
Update: if we are getting rid of `containerproject`, this can probably either go directly in the existing `project` package or as the `containerconfig` package under the existing project directory.